app/helpers/textual_mixins/textual_refresh_status.rb

```diff
@@ -1,20 +1,32 @@
 module TextualMixins::TextualRefreshStatus
+  include ActionView::Helpers::DateHelper
   def textual_refresh_status
-    last_refresh_status = @record.last_refresh_status.titleize
-    if @record.last_refresh_date
-      last_refresh_date = time_ago_in_words(@record.last_refresh_date.in_time_zone(Time.zone)).titleize
+    refresh_status(@record)
+  end
+
+  def textual_refresh_date
+    refresh_date(@record)
+  end
+
+  def refresh_status(record)
+    last_refresh_status = record.last_refresh_status&.titleize
+    refresh_time = record.last_refresh_date
+    if refresh_time
+      last_refresh_date = refresh_time ? time_ago_in_words(refresh_time).titleize : _(Never)
```

Comment:
```suggestion
      last_refresh_date = refresh_time ? time_ago_in_words(refresh_time).titleize : _("Never")
```

This feels like it should have blown up in test.

Reply:
done, fixed it.